Ryujinx.HLE/HOS/Services/Time/TimeZone/TimeZoneManager.cs

```diff
@@ -0,0 +1,292 @@
+﻿using LibHac.Fs.NcaUtils;
+using Ryujinx.Common.Logging;
+using Ryujinx.HLE.FileSystem;
+using System;
+using System.Collections.ObjectModel;
+using LibHac.Fs;
+using System.IO;
+using System.Collections.Generic;
+using TimeZoneConverter.Posix;
+using TimeZoneConverter;
+
+using static Ryujinx.HLE.HOS.Services.Time.TimeZoneRule;
+using static Ryujinx.HLE.HOS.ErrorCode;
+
+namespace Ryujinx.HLE.HOS.Services.Time.TimeZone
+{
+    public sealed class TimeZoneManager
+    {
+        private const long TimeZoneBinaryTitleId = 0x010000000000080E;
+
+        private static TimeZoneManager instance;
+
+        private static object instanceLock = new object();
+
+        private Switch         _device;
+        private TimeZoneRule   _myRules;
```

Comment:
```suggestion
        private TimeZoneRule _myRules;
```
And then align other vars names